apps/seg/itkimage2segimage.cxx

```diff
@@ -43,12 +45,15 @@ int main(int argc, char *argv[])
   if(dicomDirectory.size()){
     OFList<OFString> fileList;
     cout << "Searching recursively " << dicomDirectory << " for DICOM files" << endl;
+#if _WIN32
+    replace(dicomDirectory.begin(), dicomDirectory.end(), '/', '\\');
```

Comment:
Correction, I misunderstood this code. OS-independent definition of file separator won't help here, but I still wonder if this code can be improved with some DCMTK functionality to avoid `#_WIN32`. @jriesmeier how does DCMTK handle this situation?